### ISSUE TITLE
Invizimals not work correctly

### ISSUE THREAD

**Issue by Dokman** [GamerROMInc/Adrenaline-Mod#2]
To reproduce the issue i started a new game then i tried to capture the first invizimal called ironbuck but instead of beig crushed it doesn't do nothing

**Comment by GamerROMInc** [comment]
What version of the adrenaline mod are you using?

**Comment by Dokman** [comment]
your latest version

**Comment by GamerROMInc** [comment]
I just updated the app to v1.02 changelog and update is live: https://www.psx-place.com/resources/adrenaline-mod-v7-orbispsp-os-edition.1138/update?update=1960 hopefully this fixes your issue if it doesnt it maybe a game compatability issue

**Comment by Dokman** [comment]
then it's a game compatibility issue cause i am at 1.02

**Comment by GamerROMInc** [comment]
I agree tried playing thrilleville has video issues as well, so hipefully this issue is solved would you mind to close this or would tou orefere to keep it open?
